web/exercise-05/details.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pokemons Info</title>
+    <style>
+        body {
+            font-family: sans-serif;
+        }
+
+        p {
+            padding-bottom: 0.5rem;
+            border-bottom: 2px solid #ccc;
+        }
+
+        p:nth-of-type(4), p:last-of-type{
+            display: inline-block;
+            border: none;
+        }
+
+        #show_more {
+            background-color: antiquewhite;
+            padding: 0.5rem;
+            cursor: pointer;
+        }
+
+        @media only screen and (max-width: 600px) {
+            body {
+                font-size: 70%;
+            }
+        }
```

Comment:
can we handle the iframe responsiveness on the page in a better way when the window size is decreased.